.github/workflows/check-wasm.yml

```diff
@@ -19,20 +19,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: set up rust
-        uses: dtolnay/rust-toolchain@master
-        id: toolchain
+      - name: Install rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          target: wasm32-unknown-unknown
-          components: rust-src
-          toolchain: nightly-2024-01-01
+          rustflags: ""
```

Comment:
Alas we should write 'rustflags: ""` everywhere. Otherwise our optimization rust flags will be replaced by `-D warnings`. Strange behavior of this action tbh